### PR TITLE
Javascript unary expression

### PR DESCRIPTION
**Pull request by rockerBOO**
Move unary expressions to operator.

https://tc39.es/ecma262/#sec-conditional-operator

2020-11-27-103720_4480x1440_scrot

[old punctuation.special screenshot](https://user-images.githubusercontent.com/15027/100382449-65fe9280-2fe9-11eb-8834-6da0b7f550b1.png)

### CODE REVIEW

File: queries/javascript/highlights.scm [hunk 1]
```diff
@@ -184,6 +184,7 @@
 
 (binary_expression "/" @operator)
 (ternary_expression ["?" ":"] @operator)
+(unary_expression) @punctuation.special
```
Comment (vigoux) [inline queries/javascript/highlights.scm:187]: Is it not an operator ?

Comment (rockerBOO) [reply queries/javascript/highlights.scm:187]: https://tc39.es/ecma262/#sec-unary-operators
 
Looks like they would be operators.

File: queries/javascript/highlights.scm [hunk 2]
```diff
@@ -184,6 +184,7 @@
 
 (binary_expression "/" @operator)
 (ternary_expression ["?" ":"] @operator)
+(unary_expression) @operator
```
Comment (steelsojka) [inline queries/javascript/highlights.scm:187]: This is incorrect. The unary expression node covers the indentifier argument as well. This should be `(unary_expression "!") @operator`, matching the anonymous node.

**Review by steelsojka — CHANGES_REQUESTED** [review]

**Comment by rockerBOO** [comment]
Now should mark all the operators specifically. 

2020-11-27-115422_4480x1440_scrot

**Review by steelsojka — APPROVED** [review]

**Pull request merged**